resources/js/app.js

```diff
@@ -990,9 +990,14 @@ function initAutocomplete() {
     } else {
       alert('You must consent to the use of your data in order to register');
     }
-
   });
 
+  $('#login-form-submit').on('click', function(e) {
+    // We've seen double submits of the login form, leading to 419 errors.  Prevent the user submitting twice by
+    // double-clicking.
+    $('#login-form-submit').attr('disabled', 'disabled')
```

Comment:
I've noticed in the past that disabled fields don't appear in POST data as a result of being disabled.  Assuming it's not a problem here though, as we won't be checking on the presence of this particular field.